### DISCUSSION TITLE
Erro ao adicionar na interface o IValidatable

### DISCUSSION THREAD

**Discussion by marcionevesdev** [balta-io/7196#23] [Perguntas e Respostas] [answered]
Boa tarde.  eu adicionei a lib Flunt mas  meu arquivo ICommands nao esta aceitando  IValidatable 

Segue o código da página:

-----------------------------------------
using Flunt.Validations;

namespace Todo.Domain.Commands.Contracts
{
    public interface ICommand : IValidatable
    {

    }
}
----------------------------
Desde ja agradeço

**Comment by capneiva** [comment]
Bom dia!

Tive o mesmo problema... Testando o projeto 7196, onde há referência de IValidatable, vi que estava usando a versão do Frunt 1.0.4.

**Reply by capneiva** [reply]
A última versão que ele instala é a 2.0.5, então removi o package,  dotnet remove package Flunt e instalei novamente com a versão 1.0.4,  dotnet add package Flunt --version 1.0.4. Com isso, funcionou...

**Comment by andrebaltieri** [comment, marked as answer]
@marcioLegiaodeAco @capneiva como estão?
Por favor, usem as mesmas versões do curso primeiro... depois que se familiarizarem com o pacote, podem atualizar para versão mais recente seguindo a documentação do mesmo!

**Comment by marcionevesdev** [comment]
Instalei a versão 1.0.4 e funcionou ! Obrigado a todos !!
